src/main/java/io/temporal/internal/metrics/MetricsType.java

```diff
@@ -122,32 +77,26 @@
       TEMPORAL_METRICS_PREFIX + "local-activity-panic";
   public static final String LOCAL_ACTIVITY_EXECUTION_LATENCY =
       TEMPORAL_METRICS_PREFIX + "local-activity-execution-latency";
-  public static final String WORKER_PANIC_COUNTER = TEMPORAL_METRICS_PREFIX + "worker-panic";
-
-  public static final String TASK_QUEUE_QUEUE_LATENCY =
-      TEMPORAL_METRICS_PREFIX + "taskqueue-queue-latency";
-
-  public static final String UNHANDLED_SIGNALS_COUNTER =
-      TEMPORAL_METRICS_PREFIX + "unhandled-signals";
   public static final String CORRUPTED_SIGNALS_COUNTER =
       TEMPORAL_METRICS_PREFIX + "corrupted-signals";
 
   public static final String WORKER_START_COUNTER = TEMPORAL_METRICS_PREFIX + "worker-start";
   public static final String POLLER_START_COUNTER = TEMPORAL_METRICS_PREFIX + "poller-start";
 
   public static final String TEMPORAL_REQUEST = TEMPORAL_METRICS_PREFIX + "request";
-  public static final String TEMPORAL_ERROR = TEMPORAL_METRICS_PREFIX + "error";
-  public static final String TEMPORAL_LATENCY = TEMPORAL_METRICS_PREFIX + "latency";
-  public static final String TEMPORAL_INVALID_REQUEST = TEMPORAL_METRICS_PREFIX + "invalid-request";
+  public static final String TEMPORAL_REQUEST_FAILURE = TEMPORAL_REQUEST + "-failure";
```

Comment:
Looks like we are deviating from convention we use from go-sdk.  Is there a reason to move away from existing convention.  We don't want to build 2 separate dashboards for each different type of sdk.